vowpalwabbit/CMakeLists.txt

```diff
@@ -288,7 +298,7 @@ target_link_libraries(vw
   PUBLIC
     VowpalWabbit::explore VowpalWabbit::allreduce Boost::boost
   PRIVATE
-    Boost::program_options ${CMAKE_DL_LIBS} ${LINK_THREADS} vw_io
+    Boost::program_options ${CMAKE_DL_LIBS} ${LINK_THREADS} vw_io ZLIB::ZLIB
```

Comment:
Just reminding you that you said you probably can remove `zlib` from here

Reply:
Thanks! I may have forgotten... :P